### PR TITLE
Update base_import_async.py

### PR DESCRIPTION
**Pull request by OdooMod**
fix error,python csv "delimiter" must be string, not unicode

### CODE REVIEW

**Comment by lmignon** [comment]
This one supersedes #23
@waveyeung It seems that you commit is not complete (https://github.com/OCA/connector-interfaces/pull/23/files#diff-ab9d816d577d939a66940f1c6532b2a3R82)

**Comment by OdooMod** [comment]
@Imigonon Error msg at travis-ci is "ImportError: No module named connector"?

**Comment by lmignon** [comment]
@waveyeung The travis config seems to no more works with the lastest MQT.
Can you remove the line 28 in the file .travis.yml 

```
  - git clone https://github.com/OCA/connector -b ${VERSION} $HOME/connector
```

https://github.com/OCA/connector-interfaces/blob/8.0/.travis.yml#L28 and add an new file named oca_dependencies.txt next to .travis.yml with the following content

```
connector
```

You can find a sample file here https://github.com/OCA/maintainer-quality-tools/blob/master/sample_files/oca_dependencies.txt

**Comment by lmignon** [comment]
@waveyeung You should include all the changes of others PR #27 #26 #25 into this one.

File: .travis.yml [hunk 1]
```diff
@@ -25,7 +25,7 @@ install:
   - git clone https://github.com/OCA/maintainer-quality-tools.git ${HOME}/maintainer-quality-tools
   - export PATH=${HOME}/maintainer-quality-tools/travis:${PATH}
   - travis_install_nightly
-  - git clone https://github.com/OCA/connector -b ${VERSION} $HOME/connector
+  #- git clone https://github.com/OCA/connector -b ${VERSION} $HOME/connector
```
Comment (lmignon) [inline .travis.yml]: you can completely remove the line

**Comment by OdooMod** [comment]
@lmignon OK. Thanks for help, that's interesting. :)

**Comment by lmignon** [comment]
You are welcome. Thanks to you for contributing. 
Nevertheless the changes in base_import_async/models/base_import_async.py seem to be incomplete compared to the PR #23 (https://github.com/OCA/connector-interfaces/pull/23/files#diff-ab9d816d577d939a66940f1c6532b2a3R82)

**Comment by lmignon** [comment]
LGTM :+1: 

For the record you already have one :+1:  from @rvalyi on #23

**Comment by sbidoul** [comment]
:+1:

**Pull request merged**